src/services/jobs/dto/job.model.ts

```diff
@@ -0,0 +1,32 @@
+import { Field, Float, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class jobModel {
```

Comment:
this DTO looks similar to the `create job` DTO, so why do you need this DTO?

Reply:
I forgot to put the id field, which is the main difference, in this DTO we have the ID of the job, which is something we don't have in the CreateJob Input DTO